app/main.py

```diff
@@ -2,0 +1,19 @@
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split(" ")
+    file_to_move = command[1]
+    destination = command[2].split("/")
+    if file_to_move != "".join(destination):
```

Comment:
This condition doesn't seem to be correct. You're comparing the filename with the joined destination path. It might not work as expected if the filename contains directories. (Bug)